From 5703ff612a6658dd73513891a87ef3704a1ed9fa Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Tue, 11 Jan 2022 11:58:58 +0000
Subject: [PATCH] gstreamer: Use https

The unauthenticated git protocol is no longer supported by github
so we should use https for do_fetch stages.

Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 .../gstreamer/gstreamer1.0-plugins-bad_1.16.3.bbappend            | 8 ++++----
 .../gstreamer/gstreamer1.0-plugins-good_1.16.3.bbappend           | 8 +++++++-
 2 files changed, 11 insertions(+), 5 deletions(-)

diff --git a/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.3.bbappend b/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.3.bbappend
index 22722af..45b61cc 100644
--- a/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.3.bbappend
+++ b/recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.3.bbappend
@@ -3,7 +3,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}/:"
 SRC_URI_remove = "https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-${PV}.tar.xz"
 
 SRC_URI_append = " \
-    gitsm://github.com/renesas-rcar/gst-plugins-bad.git;branch=RCAR-GEN3e/1.16.3;name=base \
+    git://github.com/renesas-rcar/gst-plugins-bad.git;protocol=https;branch=RCAR-GEN3e/1.16.3;name=base \
     file://0001_fix_waylandsink_fullscreen.patch \
     file://0002-waylandsink-Add-set-window-position.patch \
     file://0003-waylandsink-Add-property-out-w-out-h-to-display-expe.patch \
@@ -17,9 +17,9 @@ DEPENDS += "weston"
 S = "${WORKDIR}/git"
 
 do_configure_prepend() {
-    cd ${S}
-    ./autogen.sh --noconfigure
-    cd ${B}
+	cd ${S}
+	./autogen.sh --noconfigure
+	cd ${B}
 }
 
 PACKAGECONFIG_append = " faac faad"
diff --git a/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.16.3.bbappend b/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.16.3.bbappend
index 5dcc8d8..a60bba2 100644
--- a/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.16.3.bbappend
+++ b/recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.16.3.bbappend
@@ -2,7 +2,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}/:"
 
 SRC_URI_remove = "https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-${PV}.tar.xz"
 SRC_URI_append = " \
-    gitsm://github.com/renesas-rcar/gst-plugins-good.git;branch=RCAR-GEN3e/1.16.3;name=base \
+    git://github.com/renesas-rcar/gst-plugins-good.git;protocol=https;branch=RCAR-GEN3e/1.16.3;name=base \
 "
 
 SRCREV_base = "34a8bf145506eca37291c8e72ab09fa0b5eb6149"
@@ -11,6 +11,12 @@ DEPENDS += "mmngrbuf-user-module"
 
 S = "${WORKDIR}/git"
 
+do_configure_prepend() {
+    cd ${S}
+    ./autogen.sh --noconfigure
+    cd ${B}
+}
+
 EXTRA_OEMESON_append = " \
     -Dignore-fps-of-video-standard=true \
 "
-- 
2.7.4

